react-search/my-app/src/List.tsx

```diff
@@ -4,14 +4,14 @@ type Props = {
 };
 
 export function List({ quotes, searchTerm }: Props) {
-  const pElements = [];
-  for (let i = 0; i < quotes.length; i++) {
-    if (quotes[i].toLowerCase().match(searchTerm.toLowerCase())) {
-      pElements.push(<p key={i}>{quotes[i]}</p>);
-    }
-  }
-  if (pElements.length === 0) {
-    pElements.push(<p>No items match the filter.</p>);
-  }
-  return <>{pElements} </>;
+  const matchedStrings = quotes.filter(
+    (str) => !!str.toLowerCase().match(searchTerm.toLowerCase())
```

Comment:
`filter` will include the string if the expression is truthy. The `!!` paradigm is necessary only when you need an actual boolean (`true` or `false`). So they're not needed here.

Reply:
Not sure how/why I put that in there. Thanks for catching it!